### DISCUSSION TITLE
Localization not working for my Theia extension

### DISCUSSION THREAD

**Discussion by BarryDrez** [eclipse-theia/theia#12258] [Q&A]
I recently upgraded my extension from Theia 1.25.0 to 1.34.0.  Under 1.25.0, my localization was working fine; however, it stopped working when I upgraded.  I traced it back to Theia 1.27 where it breaks for me.  Theia text is displayed but not the text I have in my resource bundle.  It is not obvious to me what I need to do to fix this.  There is nothing in the logs.  The one thing I do notice is that when nls.localize(key: string, defaultValue: string, ...args: FormatType[]) is called, localization is undefined.  Line 33: if (localization) {
so it drops through and returns the defaultValue.

Any ideas as to how I can correct this?

**Comment by msujew** [comment]
Weird, there's nothing in 1.27 that would indicate such a change in behavior. Do you have a reproducible sample that I could use to figure out what's wrong?

**Reply by BarryDrez** [reply]
@msujew Thanks for the reply.  Sorry for the confusion.  What I meant to say was that 1.27 was good, and it broke under 1.28.  I will try to get a scaled down version of my application for you asap.
